static/js/components/VideoPlayer.js

```diff
@@ -339,7 +337,12 @@ class VideoPlayer extends React.Component<*, void> {
           })
         })
         if (this.tech_.hls !== undefined) {
-          this.tech_.hls.selectPlaylist = selectPlaylist
+          const _originalSelectPlaylist = this.tech_.hls.selectPlaylist
```

Comment:
This was overriden in a previous PR so that the size of the video in the browser is ignored when determining the optimal resolution to play.  Can you revert to using that custom algorithm instead of the default one?

Reply:
Ah, I see what you mean.

Updated to use a modified version of the previous custom algorithm. The modification is: the algorithm will now select the highest *active* playlist that is less than the available system bandwidth, or the first active playlist if no active playlist is <= system bandwidth.

The quality selector widget sets all non-selected playlists to disabled. If 'auto' is selected it sets all playlists to active.